pyanaconda/ui/lib/software.py

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (C) 2023 Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+from collections import namedtuple
+from blivet.arch import is_arm
+from pyanaconda.core.i18n import _
+
+FEATURE_64K = "64k"
+KernelFeatures = namedtuple("KernelFeatures", "page_size_64k")
+
+def get_display_kernel_features(payload):
+    """Returns a dictionary with that shows which kernels should be shown in the UI.
+    """
+    features = {
+        FEATURE_64K: is_arm() and any(payload.match_available_packages("kernel-64k"))
+    }
+
+    return features
```

Comment:
This name is a bit cryptic to me. Wouldn't it be better to use name what you have when you storing this value `get_available_kernel_features`?